tracing-attributes/src/lib.rs

```diff
@@ -294,6 +305,18 @@ mod expand;
 ///     Ok(())
 /// }
 /// ```
+/// 
+/// If you're returning a [`std::result::Result<T, E>`] and `E` implements [`std::error::Error`], it can be
```

Comment:
docs nit: generally, we prefer to avoid second-person ("you") language in the documentation. also, I don't think it's necessary to use the fully qualified path for `Result`:
```suggestion
/// If a function returns a [`Result<T, E>`] and `E` implements [`std::error::Error`], it can be
```